code/modules/instruments/songs/_song.dm

```diff
@@ -211,7 +211,8 @@
 	//we can not afford to runtime, since we are going to be doing sound channel reservations and if we runtime it means we have a channel allocation leak.
 	//wrap the rest of the stuff to ensure stop_playing() is called.
 	do_hearcheck()
-	SEND_SIGNAL(parent, COMSIG_SONG_START)
+	SEND_SIGNAL(parent, COMSIG_SONG_START, src)
+	SEND_SIGNAL(user, COMSIG_SONG_START, src)
```

Comment:
Surely these shouldn't be the same signal? `COMSIG_INSTRUMENT_SONG_START` and `COMSIG_MOB_START_PLAYING_SONG` or something